Manifest.toml

```diff
@@ -1,8 +1,8 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.9.0"
+julia_version = "1.8.5"
```

Comment:
Is the downgrade to 1.8 necessary to make this work? Unfortunately, we have to have the full manifest in the repo because of the reliance on unregistered packages.